### PR TITLE
Make transformation RNG less nasty

### PR DESCRIPTION
**Pull request by Stadler76**
I've created the function `MutationsHelper.tfChance(min:int, max:int):Boolean` as a wrapper around `rand(x) === 0` for TF classes and functions.
It has a min and max value to make it more likely to have at least one mutation, when consuming transformatives.
I've used Reptilum as an example, but this could/should always be ported to other transformatives to make the RNG less nasty.

Changes in Reptilum are:
- Replaced `rand(3) === 0` with `tfChance(2, 3)`
- Replaced `rand(4) === 0` with `tfChance(2, 4)`
- Replaced `rand(5) === 0` with `tfChance(3, 5)`
In other words: its always min = the old chance - 2 (the minimum value being 2) and max = the old chance.

Note, that tfChance is not failsafe, so please make sure, that min is always smaller, than max. And with 'always smaller' I mean: Don't replace for example `rand(2) === 0` with `tfChance(2, 2)`, because that would be quite stupid ...

### CODE REVIEW

**Comment by brrritssocold** [comment]
Was also looking into this, but with an interface that can be injected instead of a Util function.
Reason being that any code using RNG is royal pain to test.

Maybe build something that is more intuitive and generic? Say `chancePercent(30)` for a 30% chance for getting a `true`.

**Comment by Stadler76** [comment]
@brrritssocold wrote:
> Was also looking into this, but with an interface that can be injected instead of a Util function.
> Reason being that any code using RNG is royal pain to test.
> 
> Maybe build something that is more intuitive and generic? Say `chancePercent(30)` for a 30% chance for getting a `true`.

Well, `tfChance` depends on 
```as3
public var changes:int = 0;
```
in `MutationsHelper.as`

If you could take account for that, then go on.

Probably you should allow omitting the max value, so ppl. could use something, like `tfChance(2)` as a simple replacement for `rand(2) === 0`?
I'll leave the code as is for now.

**Comment by Stadler76** [comment]
@brrritssocold: Would PR #1246 be a step towards that?
